alternator/executor.cc

```diff
@@ -864,6 +877,9 @@ static future<executor::request_return_type> create_table_on_shard0(tracing::tra
     // (e.g., verify that this table doesn't already exist) - we can only
     // do this further down - after taking group0_guard.
     std::string table_name = get_table_name(request);
+    // Validate table name and raise exception if `table_name` is invalid.
```

Comment:
I'd shorten the comment here to not repeat the function name (it's the same information basically)

Reply:
Done!